inc/saihostintf.h

```diff
@@ -692,6 +597,179 @@ typedef sai_status_t(*sai_get_hostif_attribute_fn)(
+    * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+    */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE = SAI_HOSTIF_ATTR_START,
+
+    /**
+    * @brief Host interface table entry match field object-id
+    *
+    * Valid only when #SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == #SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_PORT
+    * || #SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_LAG || #SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_VLAN
+    * should be port object when type is SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_PORT
+    * should be lag object when type is SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_LAG
+    * should be VLAN ID object when type is SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_VLAN
+    *
+    * @type sai_object_id_t
+    * @objects SAI_OBJECT_TYPE_PORT, SAI_OBJECT_TYPE_LAG, SAI_OBJECT_TYPE_ROUTER_INTERFACE
+    * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+    * @condition SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_PORT or SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_VLAN or SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_LAG
+    */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_OBJ_ID,
+
+    /**
+    * @brief Host interface table entry match field trap-id
+    *
+    * Valid only when #SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == #SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_PORT || 
+    * #SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_LAG || #SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_VLAN ||
+    * #SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_TRAP_ID
+    * @type sai_object_id_t
+    * @objects SAI_OBJECT_TYPE_HOSTIF_TRAP
+    * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+    * @condition SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_PORT or SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_VLAN or SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_LAG or SAI_HOSTIF_TABLE_ENTRY_ATTR_TYPE == SAI_HOST_INTERFACE_TABLE_ENTRY_TYPE_TRAP_ID
+    */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_TRAP_ID,
+
+    /**
+    * @brief Host interface table entry action channel
+    *
+    * @type sai_host_interface_table_entry_channel_type_t
+    * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+    */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_CHANNEL,
+
+    /**
+    * @brief Host interface table entry action target host interface object
+    *
+    * Valid only when #SAI_HOSTIF_TABLE_ENTRY_ATTR_CHANNEL = #SAI_HOST_INTERFACE_TABLE_ENTRY_CHANNEL_TYPE_FD
+    * @type sai_object_id_t
+    * @objects SAI_OBJECT_TYPE_HOSTIF
+    * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+    * @condition SAI_HOSTIF_TABLE_ENTRY_ATTR_CHANNEL == SAI_HOST_INTERFACE_TABLE_ENTRY_CHANNEL_TYPE_FD
+    */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_HOST_IF,
+
+    /**
+    * @brief Rnd of attributes
+    */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_END,
+
+    /** Custom range base value */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_CUSTOM_RANGE_START = 0x10000000,
+
+    /** End of custom range base */
+    SAI_HOSTIF_TABLE_ENTRY_ATTR_CUSTOM_RANGE_END
+
+} sai_hostif_table_entry_attr_t;
```

Comment:
@itaibaz ideally it should be towards the better match, in the above the left is highest priority or rightmost.